pkg/sql/row/fetcher_mvcc_test.go

```diff
@@ -108,11 +108,13 @@ func TestRowFetcherMVCCMetadata(t *testing.T) {
 	}
 	var rf row.Fetcher
 	if err := rf.Init(
+		ctx,
 		keys.SystemSQLCodec,
-		false, /* reverse */
+		false, /*reverse*/
 		descpb.ScanLockingStrength_FOR_NONE,
-		true, /* isCheck */
+		true, /*isCheck*/
```

Comment:
nit: missing spaces

Reply:
but everyone else in this file doesn't use spaces... it's a mess. I'm going to leave this.

Reply:
Wait, this file uses spaces, but the others don't? These comments are driving me crazy, and it doesn't matter so I'll still leave as such I think